Dockerfile

```
@@ -1,8 +1,8 @@
 FROM python:3.11
 ARG ANSIBLE_VERSION=stable-2.16
 RUN pip install --no-cache-dir --disable-pip-version-check https://github.com/ansible/ansible/archive/${ANSIBLE_VERSION}.tar.gz
-COPY ./requirements.txt /tmp/requirements.txt
-RUN pip install --no-cache-dir --disable-pip-version-check --requirement /tmp/requirements.txt
+COPY ./requirements.txt ./requirements-dev.txt /tmp/
+RUN sh -c 'for req in /tmp/requirements*.txt; do pip install --no-cache-dir --disable-pip-version-check -r $req; done'
```

can we do this instead of loop ?

```suggestion
RUN pip install --no-cache-dir --disable-pip-version-check --requirement /tmp/requirements.txt --requirement /tmp/requirements-dev.txt
```